packages/field-views-loader/index.js

```diff
@@ -1,10 +1,9 @@
 const loaderUtils = require('loader-utils');
-const endent = require('endent');
 
 function serialize(value, allPaths) {
   if (typeof value === 'string') {
     allPaths.add(value);
-    return `loaders['${value}']`;
+    return `loaders[${JSON.stringify(value)}]`;
```

Comment:
The paths need to be JSON.stringified since windows paths have `\` which need to be escaped